library/ws-widget/collection/src/lib/_common/content-toc/app-toc-about/app-toc-about.component.html

```diff
@@ -358,7 +358,7 @@
     </ng-container>
 
     <ng-container *ngIf="!skeletonLoader">
-        <div class="flex flex-column gap-3" [ngClass]="{'hideAbove1200': !router.url.includes('/viewer/pdf/do_')}" >
+        <div class="flex flex-column gap-3" [ngClass]="{'hideAbove1200': !router.url.includes('/viewer/')}" >
```

Comment:
make sure it will not impact other areas

Reply:
It won't! Before it was not working for the viewer page (only for PDF related was working), not it will work for all format